crates/bevy_ui/src/render/mod.rs

```diff
@@ -698,7 +698,7 @@ pub fn extract_uinode_text(
         };
 
         // Skip if not visible or if size is set to zero (e.g. when a parent is set to `Display::None`)
-        if !view_visibility.get() || uinode.size().x == 0. || uinode.size().y == 0. {
+        if !view_visibility.get() || uinode.is_empty() {
```

Comment:
I assume the change in behaviour to nodes with negative extents is intentional?

Reply:
Taffy clamps the layout size output to zero or greater, so there shouldn't be any difference between `==` or `<=`. Using `<=` in the `is_empty` seems a safer choice though.